mne_bids/read.py

```diff
@@ -415,17 +429,30 @@ def read_raw_bids(bids_fname, bids_root, extra_params=None,
             coordsystem_json = json.load(fin)
 
         # Get coordinate frames that electrode coordinates are in
+        # Note: all coordinate frames should be lower-case
         if kind == "meg":
-            coord_frame = coordsystem_json['MEGCoordinateSystem']
+            coord_frame = coordsystem_json['MEGCoordinateSystem'].lower()
+            coord_unit = coordsystem_json['MEGCoordinateUnits']
         elif kind == "ieeg":
-            coord_frame = coordsystem_json['iEEGCoordinateSystem']
-        else:
+            coord_frame = coordsystem_json['iEEGCoordinateSystem'].lower()
+            coord_unit = coordsystem_json['iEEGCoordinateUnits']
+
+            # default coordinate frames to available ones in mne-python
+            # noqa: see https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html
+            if coord_frame not in _ieeg_coordinate_frames_dict:
```

Comment:
I'm a bit confused. There are only two coordinate systems allowed in the BIDS specification (in the link above) -- Pixels and ACPC. Why does your dictionary not contain them? Furthermore, I see that there are some coordinate systems that are common to MEG/EEG/iEEG and some that are are separate for each. Finally, you should put `_ieeg_coordinate_frames_dict` in the `config.py` file like the rest of these variables.